pika/spec.py

```diff
@@ -317,7 +317,7 @@ def encode(self):
             data.encode_short_string(pieces, self.capabilities)
             bit_buffer = 0
             if self.insist:
-                bit_buffer = bit_buffer | (1 << 0)
+                bit_buffer |= 1 << 0
```

Comment:
`pika/spec.py` is auto-generated. Please re-generate this file using the changes you made in `utils/codegen.py`. Thanks.